scripts/bootstrap

```diff
@@ -0,0 +1,5 @@
+#!/bin/bash
+yarn install
+
+git submodule init
```

Comment:
Does this project use git submodules? If it doesn't can these lines be removed please?